docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -361,100 +310,110 @@ You should be able to move the custom styles to the root class key.
   +<AddIcon htmlColor="#fff" />
   ```
 
-### Node
 
-- [Drop official node 6 support](https://github.com/nodejs/Release/blob/eb91c94681ea968a69bf4a4fe85c656ed44263b3/README.md#release-schedule), you should upgrade to node 8.
+### Tabs
 
-### UMD
+- [Tab] Remove the `labelContainer`, `label` and `labelWrapped` class keys for simplicity.
+This has allowed us to remove 2 intermediary DOM elements.
+You should be able to move the custom styles to the `root` class key.
 
-- This change eases the use of Material-UI with a CDN:
+  ![capture d ecran 2019-02-23 a 15 46 48](https://user-images.githubusercontent.com/3165635/53287870-53a35500-3782-11e9-9431-2d1a14a41be0.png)
```

Comment:
Since `capture d ecran 2019-02-23 a 15 46 48` becomes the alt text, it should probably describe the image.

Reply:
I think @oliviertassinari added this. What would be a good alt-text? It just appears in the diff because I rearranged it.